packages/flutter/lib/src/services/sensitive_content.dart

```diff
@@ -0,0 +1,139 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// @docImport 'package:flutter/widgets.dart';
+library;
+
+import 'package:flutter/foundation.dart';
+
+import 'system_channels.dart';
+
+/// The level of sensitivity that content in the widget tree has.
+///
+/// {@template flutter.services.ContentSensitivity}
+/// There are only three [ContentSensitivity] levels, and these can be set via a
+/// [SensitiveContent] widget.
+///
+/// [ContentSensitivity.sensitive] is the most severe setting, and if it is set,
```

Comment:
I think most severe is correct here but what we are actually trying to convey with this prose is that it is the most important or controlling value.

Reply:
Changed "severe" wording to "prioritized". Let me know what you think